### PR TITLE
Proposals

### PR DESCRIPTION
**Pull request by oliviertassinari**

### CODE REVIEW

File: packages/material-ui/src/styles/muiStyled.js [hunk 1]
```diff
@@ -57,10 +57,9 @@ const variantsResolver = (props, styles, theme, name) => {
 const shouldForwardProp = (prop) => prop !== 'styleProps' && prop !== 'theme';
 
 const muiStyled = (tag, options, muiOptions) => {
-  const defaultStyledResolver = styled(tag, { shouldForwardProp, ...options });
+  const name = muiOptions.muiName;
+  const defaultStyledResolver = styled(tag, { shouldForwardProp, label: name, ...options });
```
Comment (oliviertassinari) [inline packages/material-ui/src/styles/muiStyled.js:61]: https://emotion.sh/docs/labels

Capture d’écran 2020-09-21 à 23 09 41

Comment (oliviertassinari) [reply packages/material-ui/src/styles/muiStyled.js:61]: I have found it by luck looking at the types https://github.com/emotion-js/emotion/blob/master/packages/styled-base/types/index.d.ts#L36.

File: packages/material-ui-styled-engine-sc/src/index.js [hunk 2]
```diff
@@ -1,2 +1,12 @@
-export * from 'styled-components';
-export { default } from 'styled-components';
+import scStyled from "styled-components";
+
+export default function styled(tag, options) {
+  if (options) {
+    return scStyled(tag).withConfig({
+      displayName: options.label,
```
Comment (oliviertassinari) [inline packages/material-ui-styled-engine-sc/src/index.js:6]: Capture d’écran 2020-09-21 à 23 12 36

File: packages/material-ui/src/styles/muiStyled.js (hunk 1 above)
Comment (mnajdova) [reply packages/material-ui/src/styles/muiStyled.js:61]: Nicee

Comment (oliviertassinari) [reply packages/material-ui/src/styles/muiStyled.js:61]: In this context, maybe we don't need to set `mui` as a prefix, but rely on the default value emotion uses.

Comment (mnajdova) [reply packages/material-ui/src/styles/muiStyled.js:61]: Makes sense, will update it

**Pull request merged**